scripts/build/generate-thunkify.js

```diff
@@ -33,8 +33,8 @@ function force({ name, type }) {
   }
 }
 
-
-let content = `/**
+[true, false].forEach(isClass => {
+  let content = `/**
```

Comment:
Can we add `// Generated by ${path.basename(__filename)}` or something like that? Then GitHub will properly hide the diffs. We should do this for all of the generated files.

Reply:
See #44. I'll follow up here after that lands.